### DISCUSSION TITLE
Filter Stream: Extended Text available?

### DISCUSSION THREAD

**Discussion by OnlyOneMoreHour** [tweepy/tweepy#1453] [Questions] [answered]
Hello

I user the Filter Stream function to get all new tweets for a specific keyword. However, all tweets are shortened and whatever I try (tweet_mode='extended', extended_tweet["full_text"], tweet.extended_text, tweet.extended_tweet) only produces errors. Is it even possible to get additional info like extended_text, info about media and attachments via the stream function?

**Comment by sumit-jaswal** [comment]
If the tweet you are getting is a retweet, it will not have an[ extended_tweet attribute.](http://docs.tweepy.org/en/latest/extended_tweets.html)
And if you could add a snippet of code where you are facing the problem, it will be a lot easier to find a solution.

And what is the exact error that you get ?

**Comment by OnlyOneMoreHour** [comment]
Ah, I see. Thank you. Gonna check that out.

**Comment by Harmon758** [comment, marked as answer]
Statuses from streaming will by default have an `extended_tweet` attribute, but only if the Tweet in question is an extended Tweet.
See [the Streaming section of the documentation on extended Tweets](https://tweepy.readthedocs.io/en/latest/extended_tweets.html#streaming).
